src/index.js

```diff
@@ -12,11 +13,11 @@ import { createLogger } from 'redux-logger';
 import promise from 'redux-promise-middleware';
 import createDebounce from 'redux-debounced';
 import * as Sentry from '@sentry/browser';
+import theme from '@/theme';
```

Comment:
I'm thinking we might have more themes in the future. What do you think about putting this in a `themes` directory with this named `default` in `default.js`?

Reply:
Great idea.

Reply:
Maybe just `light`?